cloud/amazon/rds.py

```diff
@@ -42,7 +42,7 @@
       - The type of database.  Used only when command=create.
```

Comment:
Hi, can anyone more experienced than me can handle this please?
I am not sure where and how to write this note.

Thanks.